Resfeber/View Controllers/AddTripViewController.swift

```diff
@@ -0,0 +1,159 @@
+            label.text = sectionTitles[i]
+            label.setDimensions(width: 86)
+            
+            textFields[i].font = UIFont.systemFont(ofSize: 14)
+            textFields[i].textColor = RFColor.red
+            startDateTextField.setInputViewDatePicker(target: self, selector: #selector(tapStartDateDone))
+            endDateTextField.setInputViewDatePicker(target: self, selector: #selector(tapEndDateDone))
+            
+            let hStack = UIStackView(arrangedSubviews: [spacer(width: 20), label, textFields[i], spacer(width: 20)])
+            hStack.axis = .horizontal
+            hStack.alignment = .firstBaseline
+            hStack.spacing = 4
+            verticalStackSubViews.append(hStack)
+            verticalStackSubViews.append(seperatorView())
+        }
+        
+        let stack = UIStackView(arrangedSubviews: verticalStackSubViews)
+        stack.axis = .vertical
+        stack.spacing = 10
+        return stack
+    }()
+
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        configureViews()
+        
+        tripService = TripService(managedObjectContext: coreDataStack.mainContext,
+                                  coreDataStack: coreDataStack)
+    }
+
+    // MARK: - Helpers
+    fileprivate func configureViews() {
+        view.backgroundColor = RFColor.background
+        navigationController?.navigationBar.tintColor = RFColor.red
+        navigationController?.navigationBar.setBackgroundImage(UIImage(), for: .default)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(addNewTripWasCancelled))
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .save, target: self, action: #selector(newTripWasSaved))
+        navigationItem.rightBarButtonItem?.isEnabled = true
+        title = "Add New Trip"
+        
+        view.addSubview(tripImage)
+        tripImage.anchor(top: view.safeAreaLayoutGuide.topAnchor, paddingTop: 36)
+        tripImage.centerX(inView: view)
+        
+        view.addSubview(tripInfoStackView)
+        tripInfoStackView.anchor(top: tripImage.bottomAnchor,
+                                 left: view.leftAnchor,
+                                 right: view.rightAnchor,
+                                 paddingTop: 36)
+    }
+
+    @objc func addNewTripWasCancelled() {
+        self.dismiss(animated: true, completion: nil)
+    }
+    //TODO: Add save image from image picker
+    // Save dates to CoreData
+    @objc func newTripWasSaved() {
+        let trip = tripService.addTrip(name: nameTextField.text ?? "",
+                                       image: nil,
+                                       startDate: nil,
+                                       endDate: nil)
+        print("Trip was created: \(trip)")
+        NotificationCenter.default.post(name: NSNotification.Name("load"), object: nil)
```

Comment:
I would probably hard code this notification name into an extension of `NSNotification.Name` so that you can simply access it through `.load`, and that way we're much less likely to fall trap to developer error in the future my mistyping the notification name

Reply:
I agree, extending NSNotification.Name seems like the best way to go. I've made the change and submitted.